chapters/20-proper23poles.tex

```diff
@@ -1,5 +1,51 @@
 \section{Proper (2,3)-poles}\label{ch:proper-23-poles}
 
+To explore multipoles effectively, it is best to start with the simplest ones and gradually move towards more complex ones. That's why we begin by examining the $k$-poles starting from the smallest $k$. Specifically, the smallest $k$ for which it is interesting to explore the colourability of $k$-poles arising from snarks with $k$-edge-cuts is 4.
+The~1-poles are trivially uncolourable.
+For a 2-pole to be colourable, both of its semiedges must have the same colour.
+Similarly, for a 3-pole, all three of its semiedges must have pairwise different colours. Also, the colouring properties of $4$-poles are already widely explored since their colouring properties are limited as well \cite{ChladnyFactorisation}.
+The analysis of the colouring properties of $5$-pole comes from the following ideas of P. J. Cameron, A. G. Chetwynd and J. J. Watkins \cite{Cameron1987}.
```

Comment:
"comes from the following ideas" toto je spravne aj ked tam nie je ta veta, ale len odvolanie na clanok?

Reply:
Podľa mňa hej. Tým vravíme, že nasledovné veci, čo píšeme pochádzajú od nich. My vlastne ani tak nevyužívame tú vetu ako skôr spôsob jej dokázania. A teda chce tam byť $5$-poles miesto $5$-pole.